src/main.ts

```diff
@@ -206,6 +207,8 @@ class TestReporter {
     switch (reporter) {
       case 'dart-json':
         return new DartJsonParser(options, 'dart')
+      case 'dotnet-nunit':
+        return new DotNetNunitParser(options)
```

Comment:
Maybe for consistency with DotnetTrxParser this should be named DotnetNunitParser?

Reply:
Yeah, the class should be name `DotnetNunitParser`.